APIManager.js

```diff
@@ -0,0 +1,74 @@
+//This is the class that will manage all your APIs
+
+class APIManager {
+  constructor() {
+    this.myData = {
```

Comment:
You did not need to add these empty keys below. You can use only `this.myData = {}` and it would still work